src/libraries/Microsoft.PowerFx.Core/Public/Types/DecimalType.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Text;
+using Microsoft.PowerFx.Core.Types;
+
+namespace Microsoft.PowerFx.Types
+{
+    public class DecimalType : FormulaType
```

Comment:
>DecimalType [](http://example.com/codeflow?start=17&length=11)

Can we leverage this in SQL compiler and remove this:
[https://github.com/microsoft/Power-Fx-Dataverse/blob/main/src/PowerFx.Dataverse/Types/SqlDecimalType.cs](https://github.com/microsoft/Power-Fx-Dataverse/blob/main/src/PowerFx.Dataverse/Types/SqlDecimalType.cs)

Reply:
Yes.  We should rework the SQL backend to use Decimal and then it can also support Float.  But I wanted to do that change as a separate PR, in coordination with the Dataverse team.

Reply:
Ack it's separate PR. 
I filed issues to track:
[https://github.com/microsoft/Power-Fx-Dataverse/issues/99](https://github.com/microsoft/Power-Fx-Dataverse/issues/99)
[https://github.com/microsoft/Power-Fx-Dataverse/issues/100](https://github.com/microsoft/Power-Fx-Dataverse/issues/100)